tabs/src/components/sample/QueryRunner.jsx

```diff
@@ -67,25 +69,62 @@ export function QueryRunner() {
         items: ['Key', 'Value'],
     };
 
-    const callGraph = () => {
-        // stub
-        console.log(requestType);
-        console.log(graphVersion);
-        console.log(query);
-        setResponseBody("");
-        setResponseHeaders([{ key: 'Content', items: ['Content', 'json'] }]);
-    };
+    async function callGraph() {
+        const queryParameters = query.substring(GRAPH_URL.length + graphVersion.length, query.length);
+        const url = RSC_API_URL + graphVersion + queryParameters;
+        const cleanedHeaders = {};
+        for (const i of requestHeaders) {
+            cleanedHeaders[i.items[0]] = i.items[1];
+        }
+        let requestParams = {};
+        if (requestType === requestTypes.GET) {
+            requestParams = {
+                method: requestType,
+                headers: cleanedHeaders,
+            };
+        } else {
+            requestParams = {
+                method: requestType,
+                headers: cleanedHeaders,
+                body: requestBody
+            };
+        }
+        const graphResponse = await fetch(url, requestParams);
```

Comment:
```suggestion
        let options = {
            method: requestType,
            headers: cleanedHeaders,
        };

        if (requestType !== requestTypes.GET) {
            options.body = requestBody;
        }

        const graphResponse = await fetch(url, options);
```

- Its cleaner if you assign a new key value with the condition that its not a GET call
- Its better to call it option, to stay consistent with how fetch is usually used

Reply:
ac5e50a6af607d75f2f13ce91cea9eedd89751ea